README.md

```diff
@@ -129,7 +129,7 @@ try {
 
 ## Documentation
 
-All URIs are relative to *https://api.upcloud.com/1.2*
+All URIs are relative to *https://api.upcloud.com/1.3*
```

Comment:
Maybe just this link is enough of documentation for users. Or we could have a mention `Look under docs for API descriptions` Anyways this whole table is too much for anyone to look trough